radosgw_agent/cli.py

```diff
@@ -108,6 +108,11 @@ def parse_args():
         help='sync bucket and user metadata, but not bucket contents',
         )
     parser.add_argument(
+        '--versioned',
```

Comment:
it'd be nice if we could detect this somehow so it wouldn't require an argument. could we just assume it might support versioning, and see based on the responses whether a given object actually has it enabled?